packages/compass/src/app/styles/index.less

```diff
@@ -51,22 +51,14 @@
     width: 100vw;
     height: 100vh;
     display: grid;
-    grid-template-columns: 248px auto;
+    grid-template-columns: 256px auto;
     grid-template-areas: 'sidebar content';
     pointer-events: none;
   }
 
   &-sidebar {
     grid-area: 'sidebar';
-    background: @grayDark3;
-  }
-
-  &-new-connection-button {
-    height: 48px;
-    box-sizing: content-box;
-    border-bottom: 1px solid rgba(0, 0, 0, 0.1);
-    // These styles are coupled with the actual button from compass-connections.
-    background: @grayDark2;
+    background: #F9FBF7;
```

Comment:
The .less colours don't have a variable for gray.light3. In the upcoming epic where we try and get rid of global styles to finish leafygreen we'll probably deal with it. Or when we do darkmode as this placeholder doesn't support darkmode anyway.